src/typings/linux-ca.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'linux-ca' {
+  // eslint-disable-next-line import/no-duplicates
+  import * as forge from 'node-forge';
+
+  export function getAllCerts(readSync?: boolean): Promise<string[]>;
```

Comment:
Given the bug, shouldn't this be
```suggestion
  export function getAllCerts(readSync?: boolean): Promise<string[][]>;
```